doc/Development.md

```diff
@@ -21,3 +21,13 @@ Make targets for common development tasks are:
 - `format` Perform automatic code formatting on Python code
 
 Additional tools can be found in `tools/`.
+
+## Pull requests
+
+We accept pull requests [via GitHub](https://github.com/Componolit/RecordFlux/compare). To contribute to the project, fork it under your own GitHub user and perform your changes on a topic branch. Ideally, create an issue in the upstream repository describing the problem you would like to solve and your intention to work on it. This will help us to point you to potential prior work and discuss your idea. Your branch should be named `issue_<ISSUE_NUMBER>`, e.g. `issue_694` where #694 is the ticket you created. For small (!) changes descriptive branch names without a ticket are acceptable. We usually prefix those with our GitHub username (e.g. `@senier_some_small_fix`) to be able to easily identify the originator of a branch.
+
+When submitting a pull request, your topic branch should be rebased to the current upstream `main` branch. Verify that all automatic checks performed by `make check`, `make test` and `make prove` succeed before submitting the PR. For Python code we follow and automatically enforce the coding style of [Black](https://pypi.org/project/black/). You can format your code automatically using the `make format` target on the command line. For Ada code (examples as well as generated code) please follow [our Ada style guide](https://github.com/Componolit/ada-style).
+
+We enforce 100% statement coverage for Python code using [pytest](https://pytest.org). Make sure to add relevant test cases to achieve that for your code. See `tests/README.md` and have a look at the existing test cases in the `tests` directory to get an idea of the structure of our test suite. Our Python code is also statically type-checked using [mypy](http://mypy-lang.org/). Make sure to include the required type annotations with your code.
+
```

Comment:
> See `tests/README.md`

It would be nice to have a link with a descriptive name (e.g., test documentation or test structure) to the document here.

Reply:
Changed in 7d41dbc.